docs/web-apps/automated-testing/electron.md

```diff
@@ -0,0 +1,72 @@
+---
+id: electron
+title: Electron Support on Sauce Labs [Beta]
+sidebar_label: Electron
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+[Electron](https://electronjs.org/) embeds Chromium and Node.js to enable web developers to create desktop applications. Sauce Labs is beta trialing support for automated testing with Electron-based apps on Windows 10 and 11, available to all users with VDC concurrency at US West or EU Central.
```

Comment:
Can we change this story, it feels a bit odd. Can we make it more "customer friendly"?